### DISCUSSION TITLE
[@types/ws] build error : Type 'Server' is not generic

### DISCUSSION THREAD

**Discussion by vincent-seibus** [DefinitelyTyped/DefinitelyTyped#65893] [Issues with a @types package] [answered]
With the 8.5.5 of the types/ws i have got a build error :
types/ws Type 'Server' is not generic

It is solved by downgrading to 8.5.4.

[a comment by an automated account is not shown]

**Comment by k-yle** [comment]
you may want to join the disucssion in #65501, which is the PR that introduced these breaking changes. There is also a proposal to revert those changes: #65849
